### DISCUSSION TITLE
Search from:me in:here on:yesterday

### DISCUSSION THREAD

**Discussion by larkox** [mattermost/community#81] [Practical Ideas]
The search box has great filters, like the user filter, the channel filter, the date filter... but for common searches they are not so fine to use.

Many times I find myself looking for a comment I made before, but I don't remember the channel. Or I do remember the channel, and I have it already open. For those I propose having these shorcuts like "from:me", "in:here", "on:today", "on:yesterday", and any other you might find interesting.

**Comment by Willyfrog** [comment]
we discussed this, and I think it would be a time saver for many people :)

also, ctrl+f already adds the channel you are in

**Reply by hmhealey** [reply]
This would also let us simplify a good amount of the code around that hotkey. Devin might be a bit disappointed by that though since I know he had to fight a bit to get it to add the current channel name to the search box like we currently have.

**Comment by chetanyakan** [comment]
We can use the space below the expanded search toolbar for more extensive search filters
![image](https://user-images.githubusercontent.com/35728906/123268516-b86e4600-d51b-11eb-8361-92560f757363.png)

**Comment by esethna** [comment]
We need to go to a full screen search results view to provide more real-estate for filters, we've got that design queued soon!
